### DISCUSSION TITLE
custom questions & automatic emails

### DISCUSSION THREAD

**Discussion by alemao8** [pretix/pretix#4464] [Q&A]
I have added several custom questions to be ansewered during the checkout process. Is it possible to include the answers in the automatic e-mail the customer receives after buying the tickets ?
How can I find the placeholder of these questions ?

**Comment by raphaelm** [comment]
It is not possible. The issue is that an email refers to an order, which can include to many products, each of which could have different responses to the question, so there's no useful way to render them.
